public/app/features/explore/state/reducers.ts

```diff
@@ -489,7 +489,16 @@ export const itemReducer = reducerFactory<ExploreItemState>({} as ExploreItemSta
 export const exploreReducer = (state = initialExploreState, action: HigherOrderAction): ExploreState => {
   switch (action.type) {
     case ActionTypes.SplitClose: {
```

Comment:
I think you could delete the ActionTypes.SplitClose and use this instead
```suggestion
    case splitCloseAction.type: {
```

Reply:
👍